From linux-usb-devel-bounces@lists.sourceforge.net Sat Dec  2 07:54:30 2006
From: Johannes Hoelzl <johannes.hoelzl@gmx.de>
To: linux-usb-devel@lists.sourceforge.net
Content-Type: multipart/mixed; boundary="=-MkeEW0mPdA4nBlkRFL9J"
Date: Sat, 02 Dec 2006 16:54:27 +0100
Message-Id: <1165074867.5129.9.camel@localhost.localdomain>
Mime-Version: 1.0
Subject: [linux-usb-devel] [PATCH] Add Baltech Reader ID to CP2101 driver


this patch adds the Baltech Reader ID to the list of USB IDs in the
CP2101 driver.

From: Johannes Hoelzl <johannes.hoelzl@gmx.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/serial/cp2101.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/serial/cp2101.c
+++ gregkh-2.6/drivers/usb/serial/cp2101.c
@@ -69,6 +69,7 @@ static struct usb_device_id id_table [] 
 	{ USB_DEVICE(0x10C4, 0x8218) }, /* Lipowsky Industrie Elektronik GmbH, HARP-1 */
 	{ USB_DEVICE(0x10C4, 0xEA60) }, /* Silicon Labs factory default */
 	{ USB_DEVICE(0x10C4, 0xEA61) }, /* Silicon Labs factory default */
+	{ USB_DEVICE(0x13AD, 0x9999) }, /* Baltech card reader */
 	{ USB_DEVICE(0x16D6, 0x0001) }, /* Jablotron serial interface */
 	{ } /* Terminating Entry */
 };
